plugins/woocommerce-beta-tester/woocommerce-beta-tester.php

```diff
@@ -3,15 +3,15 @@
  * Plugin Name: WooCommerce Beta Tester
  * Plugin URI: https://github.com/woocommerce/woocommerce-beta-tester
  * Description: Run bleeding edge versions of WooCommerce. This will replace your installed version of WooCommerce with the latest tagged release - use with caution, and not on production sites.
- * Version: 2.5.0
+ * Version: 2.5.1
  * Author: WooCommerce
  * Author URI: https://woocommerce.com/
  * Requires at least: 5.8
- * Tested up to: 6.0
- * WC requires at least: 6.7
- * WC tested up to: 7.0
+ * Requires PHP: 7.4
+ * Tested up to: 6.7
+ * WC requires at least: 9.4
+ * WC tested up to: 9.3
```

Comment:
I think we could have version 9.5 here, since it is recommended for beta testing that version

Reply:
Good point! Changed in 29c1cde5cecaed3fe967b812c4cd36068b835de2.